_docs/concepts/volumes.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: guide
+title: StorageOS Docs - Volumes
+anchor: concepts
+module: concepts/volumes
+---
+
+# Volumes
+
+StorageOS volumes are a logical construct which represent a writeable volume
+and exhibit standard POSIX semantics. We present volumes as mounts into
+containers via the StorageOS FUSE layer.
+
+Conceptually, StorageOS volumes have a front end presentation, which is the
+side the application sees, and a backend presentation, which is the actual
+on-disk format. Depending on the configuration, frontend and backend components
+may be on the same or different hosts.
+
+Volumes are formatted using the linux standard ext4 filesystem as standard.
+Different filesystems may be supported in the future.
+
+## On Disk Format
+
+StorageOS represents files on disk as... (fill in with Alex)
```

Comment:
StorageOS write data into blob files in the /var/lib/storageos/data directory in a binary format. StorageOS implements IO primitives to control byte to byte location in disk.
StorageOS volumes map to its own data files making the data isolated between volumes. A StorageOS volume can map to multiple blob files in the filesystem granting flexibility and performance enhancement.

(Its worth expanding, but this can be a first drop of information)